### PR TITLE
Merges aggregator changes according to upstream/master

### PR DESCRIPTION
**Pull request by Dragonn**

### CODE REVIEW

File: unirecfilter/lib/functions.c [hunk 1]
```diff
@@ -121,7 +124,18 @@ struct ast *newExpression(char *column, char *cmp, int64_t number, int is_signed
    free(cmp);
 
    if (id == UR_E_INVALID_NAME) {
-      printf("Warning: %s is not present in input format.\n", column);
+      printf("Warning: %s is not present in input format. Corresponding rule will always evaluate false.\n", column);
+      newast->id = UR_INVALID_FIELD;
+   }
```
Comment (cejkato2) [inline unirecfilter/lib/functions.c]: coding style: change to `} else if ...`

File: unirecfilter/lib/functions.c [hunk 2]
```diff
@@ -142,9 +156,14 @@ struct ast *newExpressionFP(char *column, char *cmp, double number)
    free(cmp);
 
    if (id == UR_E_INVALID_NAME) {
-      printf("Warning: %s is not present in input format.\n", column);
+      printf("Warning: %s is not present in input format. Corresponding rule will always evaluate false.\n", column);
       newast->id = UR_INVALID_FIELD;
    }
+   else if (ur_get_type(id) != UR_TYPE_FLOAT &&
```
Comment (cejkato2) [inline unirecfilter/lib/functions.c]: coding style

File: unirecfilter/lib/functions.c [hunk 3]
```diff
@@ -204,16 +228,26 @@ struct ast *newString(char *column, char *cmp, char *s)
...
    } else {
       newast->s = s;
    }
    int id = ur_get_id_by_name(column);
    if (id == UR_E_INVALID_NAME) {
-      printf("Warning: %s is not present in input format.\n", column);
+      printf("Warning: %s is not present in input format. Corresponding rule will always evaluate false.\n", column);
       newast->id = UR_INVALID_FIELD;
-   } else {
+   }
```
Comment (cejkato2) [inline unirecfilter/lib/functions.c]: coding style

File: unirecfilter/lib/functions.c [hunk 4]
```diff
@@ -204,16 +228,26 @@ struct ast *newString(char *column, char *cmp, char *s)
...
+   else if (ur_get_type(id) != UR_TYPE_STRING && 
+            ur_get_type(id) != UR_TYPE_BYTES &&
+            ur_get_type(id) != UR_TYPE_CHAR) {
+      printf("Warning: Type of %s is not string. Corresponding rule will always evaluate false.\n", column);
+      newast->id = UR_INVALID_FIELD;
+   }
```
Comment (cejkato2) [inline unirecfilter/lib/functions.c]: coding style `} else {`

**Review by cejkato2 — CHANGES_REQUESTED** [review]

File: nemea-modules.spec.in [hunk 5]
```diff
@@ -28,7 +28,11 @@ This package contains header file for liburfilter.
 %setup
 
 %build
+<<<<<<< HEAD
 ./configure --prefix=%{_prefix} --libdir=%{_libdir} --bindir=%{_bindir}/nemea --sysconfdir=%{_sysconfdir}/nemea -q --docdir=%{_docdir}/nemea-modules --includedir=%{_includedir} --enable-silent-rules --disable-repobuild;
+=======
+./configure -q --enable-silent-rules --disable-repobuild --prefix=%{_prefix} --libdir=%{_libdir} --bindir=%{_bindir}/nemea --sysconfdir=%{_sysconfdir} --docdir=%{_docdir}/nemea-modules --mandir=%{_mandir} --datadir=%{_datadir};
+>>>>>>> upstream/master
```
Comment (cejkato2) [inline nemea-modules.spec.in]: This is a bad mistake...

**Review by cejkato2 — CHANGES_REQUESTED** [review]

**Comment by Dragonn** [comment]
@cejkato2 Fixed as requested

**Review by cejkato2 — APPROVED** [review]

**Pull request merged**